src/interfaces/libpq/fe-secure-gssapi.c

```diff
@@ -197,36 +197,24 @@ pg_GSS_write(PGconn *conn, const void *ptr, size_t len)
 		output.value = NULL;
 		output.length = 0;
 
-<<<<<<< HEAD
 		/*
 		 * Create the next encrypted packet.  Any failure here is considered a
 		 * hard failure, so we return -1 even if bytes_sent > 0.
 		 */
 		major = gss_wrap(&minor, gctx, 1, GSS_C_QOP_DEFAULT,
-=======
-		/* Create the next encrypted packet */
-		major = gss_wrap(&minor, conn->gctx, 1, GSS_C_QOP_DEFAULT,
->>>>>>> 80831bcdbe80a6ca7f22105e32c2cbb54e125c4c
 						 &input, &conf_state, &output);
 		if (major != GSS_S_COMPLETE)
 		{
 			pg_GSS_error(libpq_gettext("GSSAPI wrap error"), conn, major, minor);
 			errno = EIO;		/* for lack of a better idea */
 			goto cleanup;
 		}
-<<<<<<< HEAD
 
 		if (conf_state == 0)
```

Comment:
Maybe make the condition the same as in the conflicting part (as if we get into the branch above, we will not reach this point anyway)?
```suggestion
		else if (conf_state == 0)
```

Reply:
hmmm, in this case it is better to update other branches as well, which seems not relevant here. So I drop my comment,